src/sentry/snuba/subscriptions.py

```diff
@@ -143,24 +143,28 @@ def update_snuba_subscription(subscription, old_dataset):
     return subscription
 
 
-def bulk_delete_snuba_subscriptions(subscriptions):
+def bulk_delete_snuba_subscriptions(subscriptions, disable_subscription=False):
     """
     Deletes a list of snuba query subscriptions.
     :param subscriptions: The subscriptions to delete
     :return:
     """
     for subscription in subscriptions:
         # TODO: Batch this up properly once we care about multi-project rules.
-        delete_snuba_subscription(subscription)
+        delete_snuba_subscription(subscription, disable_subscription)
 
 
-def delete_snuba_subscription(subscription):
+def delete_snuba_subscription(subscription, disable_subscription=False):
```

Comment:
Any reason to not make disabling its own function? Right now the function name says 'delete' and then a parameter makes it not delete.

Reply:
The weird bit about it is that there is both the snuba subscription and the postgres/sentry subscription. I am still deleting the snuba subscription, but disabling the postgres/sentry subscription.

Reply:
Yeah, I think it makes sense to make a disable function and have it disable the subscription and then call delete in snuba.